OpenEdXMobile/res/layout/activity_register.xml

```diff
@@ -5,26 +5,26 @@
     android:layout_height="match_parent"
     android:background="@android:color/white"
     android:orientation="vertical"
-    android:paddingBottom="8dp">
+    android:paddingBottom="@dimen/vertical_input_padding">
```

Comment:
Thats not logically correct, we're assigning an input padding to a ViewGroup which might or might not be an input based view.

You can just define a new dimen.